_data/sites.json

```diff
@@ -19588,6 +19588,16 @@
         ]
     },
 
+    {
+        "name": "Webhosting.dk",
+        "url": "https://www.webhosting.dk/DKK/deleteaccount.php",
+        "difficulty": "easy",
+        "notes": "Due to a legal requirement to retain information for the last 5 financial years, user accounts cannot be deleted even if they are no longer in use. This retention is necessary because the accounts are linked to domain names, payment details with Nets, and invoices, as mandated by law.",
```

Comment:
It's s tricky take, what happens if you try to login again, does it work? Or is the account purged immediate and just the data is kept for 5 years? If it's the latter, then I say we put it up as easy.

Lots of services can and do hold information, but many of them don't specify a period when it will be purged. In those cases we use impossible, but since this one claims the data is eventually erased, I think it's a different case.

Reply:
When trying to delete the account, a message about data needing to be stored for 5 years appears, and I cannot do anything but click "okay". The account isn't deleted, and I can log into it just fine. My guess is I'll have to wait for the 5 years to have passed, and then manually go in and re-request an account deletion. For that reason alone, I'd like to set the difficulty to "hard" 😅

Reply:
I'd say you can even set it to impossible, since deleting an account is different from retaining the data, they could do the first without doing the latter

Reply:
It's not _impossible_, just really hard as one would have to wait 5 years, which is infinitely annoying, however it can be done 🤷